python/ray/experimental/sgd/pytorch/pytorch_runner.py

```diff
@@ -177,20 +224,44 @@ def get_state(self):
             for k, v in state_dict.items():
                 state_dict[k] = v.cpu()
             cpu_state_dicts += [state_dict]
-        return {
+        return cpu_state_dicts
+
+    def _set_model_state_dicts(self, models_state_dicts):
+        for model, state_dict in zip(self.models, models_state_dicts):
+            model.load_state_dict(state_dict)
+
+    def get_state(self):
+        """Returns the state of the runner."""
+
+        state = {
             "epoch": self.epoch,
-            "models": cpu_state_dicts,
+            "models": self._get_model_state_dicts(),
             "optimizers": [opt.state_dict() for opt in self.optimizers],
             "stats": self.stats()
         }
+        if self.schedulers:
+            state.update({
+                "schedulers": [
+                    scheduler.state_dict() for scheduler in self.schedulers
+                ]
+            })
+        if self.use_fp16 and amp:
```

Comment:
Maybe comment what this check is for or factor it out into a util function? Seems like I've seen it at least twice.

Reply:
```suggestion
        # Check if fp16 is True and if NVIDIA Apex is imported.
        if self.use_fp16 and amp:
```